### DISCUSSION TITLE
Github Campux Expert applications Are Ended or On Progress?

### DISCUSSION THREAD

**Discussion by 8uraj** [community/community#62747] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

i see the application are open in February 2024
but stll august application form not showing in my become camus expert page ! ?

**Comment by Livinston-Bran** [comment, marked as answer]
Now Application is Open ! Don't Wait .....
https://education.github.com/campus_experts

**Comment by Livinston-Bran** [comment]
When is Next community meetup ?
